ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/IntegrationTestBase.java

```diff
@@ -195,6 +185,8 @@ public void setup() {
   public static void teardown() {
     clientAndServer.stop();
     runner.stop();
+    clientAndServer = null;
```

Comment:
Why set these to `null`, what difference does it make?

Reply:
That's to have the GC clear them out more promptly.  My reading suggests the GC won't clean them up during the test run otherwise.

Reply:
Correct, as mentioned above these are static variables which are considered a GC root so any objects they reference will never be GC'd.

Reply:
Fair enough